go.mod

```diff
@@ -54,11 +54,15 @@ require (
 	google.golang.org/genproto v0.0.0-20210921142501-181ce0d877f6 // indirect
 	google.golang.org/grpc v1.48.0
 	google.golang.org/protobuf v1.28.1
-	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
i hope that this wont cause any issues anywhere

Reply:
Me too :) 
I tested using devnets. I will give it another try after these commits